tests/integration/testecbug.nim

```diff
@@ -4,14 +4,10 @@ import ./marketplacesuite
 import ./nodeconfigs
 import ./hardhatconfig
 
-marketplacesuite "EC bug":
+marketplacesuite "Bug #821 - node crashes during erasure coding":
```

Comment:
We should never name anything after a bug, this is most likely the wrong place to put things altogether and should be moved.

Reply:
Yeah, I don't like it either, it completely breaks convention. This test probably belongs with API tests or whatnot. Will open an issue so we do it later.

Reply:
yeah, it should be part of the node tests in `node/testnode.nim` or similar.

Reply:
#846